ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/aaa-authentication.j2

```diff
@@ -10,6 +10,20 @@
 | Login | Serial-console | {{ aaa_authentication.login.serial_console }} |
 {%     endif %}
 
+{%     if aaa_authentication.policies is defined and aaa_authentication.policies is not none %}
+{%         if aaa_authentication.policies.on_failure_log is defined and aaa_authentication.policies.on_failure_log == true %}
+AAA Authentication on-failure log has been enabled
+{%         endif %}
+{%         if aaa_authentication.policies.on_success_log is defined and aaa_authentication.policies.on_success_log == true %}
+AAA Authentication on-success log has been enabled
+{%         endif %}
+{%         if aaa_authentication.policies.local is defined %}
+{%             if aaa_authentication.policies.local.allow_nopassword is defined and aaa_authentication.policies.local.allow_nopassword == false %}
+Policy local allow-nopassword-remote-login has been disabled.
+{%             endif %}
+{%         endif %}
+{%     endif %}
+
```

Comment:
The blank line should be enclosed to the `if` block to not append an extra blank line when code is not rendered.

Reply:
Fixed this and the others, I think what I wanted to emphasise here is to have group part of the command and then have a name for the group.

Reply:
Thanks done!